### ISSUE TITLE
Cannot upload small application-packages 

### ISSUE THREAD

**Issue by emipa606** [MSEndpointMgr/IntuneWin32App#127]
Recently any package below about 70mb fails to upload. 
`"Failed to create Win32 app, commit file request operation failed"`
Any package larger than that uploads fine, regardless if using AzCopy.

Any ideas of what it could be?
Ive started padding all uploads but that seems like a silly workaround.

**Comment by NickolajA** [comment]
I've not experienced this myself, did you say it happens regardless of using the "native" upload functionality or AzCopy?

**Comment by emipa606** [comment]
Yes, but I have not tested it recently so hopefully it was a temporary problem. I'll do some new tests and update this issue.

**Comment by NickolajA** [comment]
Appreciate it!

**Comment by emipa606** [comment]
Tested this during last week and it is still an issue. Not sure if it something in our tenant only.
Turned on the padding again today and all applications deployed ok.
